src/ucp/core/ucp_worker.c

```diff
@@ -554,13 +568,16 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
         goto err_destroy_uct_worker;
     }
 
+    events = (params->field_mask & UCP_WORKER_PARAM_FIELD_EVENTS) ?
+             params->events : (UCP_WAKEUP_TX_COMPLETION | UCP_WAKEUP_RX_AM);
```

Comment:
As a separate flag for UCP?
Currently `UCP_WAKEUP_RX_AM` includes `UCT_WAKEUP_RX_SIGNALED_AM`:
```
if (ucp_events & UCP_WAKEUP_RX_AM) {
    uct_events |= (UCT_WAKEUP_RX_AM | UCT_WAKEUP_RX_SIGNALED_AM);
 }
```